src/bmpcodec.c

```diff
@@ -814,7 +813,6 @@ gdip_read_bmp_image (void *pointer, GpImage **image, ImageSource source)
 			/* the new structure contains the ARGB masks */
 			void *p = &bmi;
 			BITMAPV4HEADER *v4 = p;
-			alpha_mask = v4->bV4AlphaMask;
```

Comment:
I'd prefer keeping this since the comment above also mentions the `ARGB masks` and it'd be a bit strange to not read the alpha value then. Which warning complains about this?

Reply:
The warning is that `alpha_mask` was assigned but never used. The comment is still accurate though - we just don't use the A value